### PR TITLE
Document a helper script to SSH into a beaker docker container

### PR DESCRIPTION
**Pull request by ekohl**
As suggested https://github.com/voxpupuli/modulesync_config/pull/324#issuecomment-285811167

### CODE REVIEW

File: moduleroot/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -98,3 +98,17 @@ If you don't want to have to recreate the virtual machine every time you
 can use `BEAKER_DESTROY=no` and `BEAKER_PROVISION=no`. On the first run you will
 at least need `BEAKER_PROVISION` set to yes (the default). The Vagrantfile
 for the created virtual machines will be in `.vagrant/beaker_vagrant_fies`.
+
+When using docker you can log in using SSH. The password is always `root`. A
+simple script can be used to easily log in:
```
Comment (rnelson0) [inline moduleroot/.github/CONTRIBUTING.md]: I'd note that `jq` is required as well. They'd find out anyway but doesn't hurt to warn ahead of time.

Comment (ekohl) [inline moduleroot/.github/CONTRIBUTING.md]: Added

**Comment by ekohl** [comment]
I mostly mirrored what beaker did but that's even better.

**Comment by ekohl** [comment]
Simplified it with the advice of @vinzent

**Pull request merged**